packages/core-transaction-pool-mem/lib/defaults.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports = {
+  enabled: true,
+  // How often to save/sync the transaction pool to a persistent storage.
+  // The number designates count of added or deleted transactions that are not in
+  // the persistent storage. When that number is reached a new sync of the pool is
+  // triggered.
+  syncInterval: 512,
+  maxTransactionsPerSender: process.env.ARK_TRANSACTION_POOL_MAX_PER_SENDER || 300,
+  whitelist: [],
+  allowedSenders: [],
+  maxTransactionsPerRequest: 200,
```

Comment:
What if I change `config.getConstants(height).block.maxTransactions` to 250, will this still work?

Reply:
@vasild as zillion pointed out, you should use the same value as the maxTransactions per block according to the network config.

Reply:
@zillionn Yes, the setting is only used on the p2p v1 endpoint. It doesn't make much sense right now either way, so we will make some changes in that regard.